src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2238,6 +2247,91 @@ public static long getMinRpcTimeout(TimeUnit unit)
                          getTruncateRpcTimeout(unit));
     }
 
+    public static Config.CQLStartTime getCQLStartTime()
+    {
+        return conf.cql_start_time;
+    }
+
+    public static void setCQLStartTime(Config.CQLStartTime value)
+    {
+        conf.cql_start_time = value;
+    }
+
+    /**
+     * How much time the item is allowed to spend in (currently only Native) queue, compared to {@link #nativeTransportIdleTimeout()},
```

Comment:
Compared to `nativeTransportIdleTimeout()` or to `native_transport_timeout`? It seems like in `Dispatcher` we have:

```
return requestExecutor.oldestTaskQueueTime() < (DatabaseDescriptor.getNativeTransportTimeout(TimeUnit.NANOSECONDS) * threshold);
```

Also, should the default threshold be `1.0` or `Double.MAX_VALUE`?

Reply:
Right; the idea was that for starters we begin with this feature effectively disabled. Keep in mind I did not know how this feature is going to be perceived since it does change many notions simultaneously, but since both Jon and Brendon have tested it and found it useful, we can go with 1.0 here and 12 seconds of deadline as defaults.